### PR TITLE
Adjust README for 2.0.0

### PR DESCRIPTION
**Pull request by IsaacMarovitz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,30 +95,12 @@ To uninstall:
...
-This fork will always stay open-source and maintained by the community.
-
-<p align="right"><a href="#top">⬆️ Back to top️</a></p>
-
 
 
 <!-- CONTACT -->
 ## Contact
 
-Lucas Lee - playcover@lucas.icu
```
Comment (Depal1) [inline README.md:119]: I think this was stated earlier. Discord is not a direct alternative to an email contact.

**Review by Depal1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IsaacMarovitz) [reply README.md:119]: Lucas is not a reliable contact. If we use an email for the contact info, it should be a different email. We could also use a Discord user id or something.

Comment (Depal1) [reply README.md:119]: I will discuss a possible email replacement or addition. A Discord username is not a replacement either. Inquiries of legal nature for instance would only be sent by email.

**Comment by amirsaam** [comment]
Images are a little bit off, bottom icon titles should have a little bit more space on the screenshots for better look.

**Comment by IsaacMarovitz** [comment]
> Images are a little bit off, bottom icon titles should have a little bit more space on the screenshots for better look.

That's the default window size

**Review by JoseMoreville — APPROVED** [review]
Looks good! please add a contact email, it could be discussed on discord, but id be good to have contact email here

**Review by Depal1 — APPROVED** [review]

**Pull request merged**